src/mpl2/test/guides1.tcl

```diff
@@ -0,0 +1,25 @@
+# Test if pin access blockage is generated correctly for a case
+# with pins in a single boundary.
```

Comment:
This comment is not related to what is being tested here. We should add a similar comment, but wrt what's the intention of this test.